frontend/test/views/AccountView.spec.js

```diff
@@ -40,32 +34,48 @@ function getMountedComponent(props) {
 }
 
 describe('AccountView.vue', () => {
-  const sandbox = sinon.createSandbox();
-  let wrapper;
+  let sandbox;
+  let mockUser;
+  let getUserStub;
 
   beforeEach(() => {
-    wrapper = getMountedComponent();
+    sandbox = sinon.createSandbox();
+    mockUser = {
+      clientSecret: 'testsecret',
+    };
+
+    getUserStub = sandbox.stub(authStore, 'getUser');
+    getUserStub.returns(mockUser);
   });
 
   afterEach(() => {
     sandbox.restore();
   });
 
   it('contains the expected content body element', () => {
+    const wrapper = getMountedComponent();
     const appHeader = wrapper.find('app-header');
     expect(appHeader.exists()).to.be.true;
 
     const appContent = wrapper.find('app-content');
     expect(appContent.exists()).to.be.true;
   });
 
+  it('should have a secretValue computed property', () => {
```

Comment:
This tests an implementation detail by verifying the computed value. I didn't comment on this test directly(links to comments about it below), but all instances of testing implementation details, such as 'computed values' aren't necessary. So this test can be removed. 

The unit tests need to verify that the DOM changes and reactions in the component are doing what operations need to be accomplished.

It was an understandable case of the situation where we couldn't get it to work earlier for the re-render at the time.  However, James followed up and figured out what was wrong, so there should be no cases within this test suite that is a direct 'expect' statement on a computed value.

https://github.com/uab-cgds-worthey/rosalution/pull/46/files/ac1f146c9e5fbcc5769bb3799c0bac96b964204f#r1124997876 , https://github.com/uab-cgds-worthey/rosalution/pull/46/files/ac1f146c9e5fbcc5769bb3799c0bac96b964204f#r1125002251